Quadriga/pom.xml

```diff
@@ -1,635 +1,676 @@
-                            </excludes>
-                        </resource>
-                    </webResources>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>buildnumber-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <phase>initialize</phase>
-                        <goals>
-                            <goal>create</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <doCheck>false</doCheck>
-                    <doUpdate>false</doUpdate>
-                    <revisionOnScmFailure></revisionOnScmFailure>
-                    <!-- <format>#{0} ({1,date})</format> -->
-                    <items>
-                        <item>buildNumber\d*</item>
-                        <item>timestamp</item>
-                    </items>
-                    <format>#{0} ({1,date,yyyy-MM-dd HH:mm:ss})</format>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.flywaydb</groupId>
-                <artifactId>flyway-maven-plugin</artifactId>
-                <version>4.0.1</version>
-                <configuration>
-                    <url>${database.url}</url>
-                    <user>${database.username}</user>
-                    <password>${database.pw}</password>
-                </configuration>
-                <dependencies>
-                    <dependency>
-                        <groupId>mysql</groupId>
-                        <artifactId>mysql-connector-java</artifactId>
-                        <version>5.1.25</version>
-                    </dependency>
-                </dependencies>
-            </plugin>
-        </plugins>
-
-        <resources>
-            <resource>
-                <filtering>true</filtering>
-                <directory>src/main/resources</directory>
-                <includes>
-                    <include>**/*</include>
-                </includes>
-            </resource>
-        </resources>
-    </build>
-
-    <scm>
-        <connection>scm:svn:http://none</connection>
-        <url>scm:svn:https://none</url>
-    </scm>
-
-    <reporting>
-        <plugins>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>cobertura-maven-plugin</artifactId>
-                <version>2.7</version>
-                <reportSets>
-                    <reportSet>
-                        <reports>
-                            <report>cobertura</report>
-                        </reports>
-                    </reportSet>
-                </reportSets>
-            </plugin>
-        </plugins>
-    </reporting>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
please keep original formatting (4 spaces indent)

Reply:
this still applies

Reply:
I have set the indent to 4 spaces in Eclipse. The formatting looks fine when I open the file in Eclipse.
![screen shot 2017-10-25 at 10 22 39 am](https://user-images.githubusercontent.com/23473237/32013187-0d23c714-b96f-11e7-8163-f89696629a62.png)

Reply:
Eclipse formatting settings:
![screen shot 2017-10-25 at 10 23 16 am](https://user-images.githubusercontent.com/23473237/32013215-1fcce184-b96f-11e7-9528-56cd6e1f0c9b.png)

Reply:
this is for java files. I think XML files are somewhere else. And as you can see in your editor, it uses tabs not spaces in the XML file.